app/controllers/member_holds_controller.rb

```diff
@@ -0,0 +1,19 @@
+class MemberHoldsController < ApplicationController
+  def create
+    if new_hold.save
+      redirect_to item_path(item), success: "Hold placed."
+    else
+      redirect_to item_path(item), error: "Something went wrong!"
+    end
+  end
+
+  private
+
+  def item
+    @item ||= Item.find(params[:item_id])
+  end
+
+  def new_hold
+    @new_hold ||= Hold.new(item: item, member: current_member, creator: current_user)
+  end
+end
```

Comment:
Not sure if it's required for the style guidelines of this project, but a few files on this PR are missing a a newline at the end.

```suggestion
end

```

Reply:
Good catch! In general we like to run things through [standardrb](https://github.com/rubyforgood/circulate#setup-pre-commit-checks) to deal with this kind of code formatting issue.